roles/edpm_ceph_hci_pre/defaults/main.yml

```diff
@@ -72,18 +72,13 @@ edpm_ceph_hci_pre_firewall_services:
     num: 122
     dport:
       - 8080
-      - 13808
+      - 8082
     ranges: "{{ edpm_ceph_hci_pre_storage_ranges | list }}"
   - name: ceph_rgw_frontend
     num: 100
     dport:
       - 8080
```

Comment:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/cifmw_cephadm/templates/ceph_rgw.yml.j2#L24

Monitor port is internal to haproxy and we don't need a firewall rule